Source/WebKit/UIProcess/Extensions/WebExtensionMatchPattern.h

```diff
@@ -79,6 +79,8 @@ class WebExtensionMatchPattern : public API::ObjectImpl<API::Object::Type::WebEx
     static URLSchemeSet& validSchemes();
     static URLSchemeSet& supportedSchemes();
 
+    static bool matchPatternSetContainsMatchForURL(const MatchPatternSet&, URL&);
```

Comment:
I'd call this (and move it up next to `static bool patternsMatchAllHosts()`:

```c++
static bool patternsMatchURL(const MatchPatternSet&, URL&);
```

Also `patternsMatchAllHosts` should use `const MatchPatternSet&` too!